cfgov/unprocessed/js/modules/FlyoutMenu.js

```diff
@@ -53,11 +53,12 @@ function FlyoutMenu( element ) { // eslint-disable-line max-statements, no-inlin
   var _isExpanded = false;
   var _isAnimating = false;
 
-  var _expandTransition;
-  var _collapseTransition;
-  var _expandTransitionMethod;
+  var _expandTransition = null;
+  var _expandTransitionMethod = null;
   var _expandTransitionMethodArgs = [];
-  var _collapseTransitionMethod;
+
+  var _collapseTransition = null;
```

Comment:
Declaring a variable by using `var a` doesn't assign it to a global singleton (window.undefined). JS is a weakly typed language so there is no implication of the type of data the variable will hold aside from naming conventions.  I would only assign variable to null when the default type expected is null i.e when querying a for a dom element that might not be found.